tests/conftest.py

```diff
@@ -29,14 +29,14 @@ def setup_test_container_props(request):
 
 
 def pytest_addoption(parser):
-    parser.addoption("--mender-client-version", required=True)
-    parser.addoption("--mender-client-deb-version", required=True)
-    parser.addoption("--mender-connect-version", required=True)
-    parser.addoption("--mender-connect-deb-version", required=True)
-    parser.addoption("--mender-configure-version", required=True)
-    parser.addoption("--mender-configure-deb-version", required=True)
-    parser.addoption("--mender-artifact-version", required=True)
-    parser.addoption("--mender-artifact-deb-version", required=True)
+    parser.addoption("--mender-client-version", required=False, default="none")
```

Comment:
Q: why `"none"` and not `None`?

Reply:
I will try :+1: It was `"none"` in some of the existing options :eyes:

Reply:
But also, shouldn't `default=None` be the same as no `default` at all? I'll remove it and see if CI complains :smiley: